kube-system/ntpd/templates/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1beta2
+kind: DaemonSet
+metadata:
+  name: {{ template "ntpd.fullname" . }}
+  labels:
+    app: {{ template "ntpd.name" . }}
```

Comment:
Please change this to  app: {{ template "ntpd.fullname" . }}. We have used this instead throughout the charts to be more descriptive with regard to the APP name :D

Reply:
Does this change throughput the entire deployment template? i.e. Line 6,14 & 19 ?